From: OpenSoar (lib/netcdf_cxx/patches/fix_new_delete.patch)
Subject: ncGroup: use delete[] for array allocated with new[]

Upstream-Status: Pending

 cxx4/ncGroup.cpp | 2 +- 
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/cxx4/ncGroup.cpp b/cxx4/ncGroup.cpp
index 1680129..2b7e94e 100644
--- a/cxx4/ncGroup.cpp
+++ b/cxx4/ncGroup.cpp
@@ -104,7 +104,7 @@ string NcGroup::getName(bool fullName) const {
     char* charName= new char[lenp+1];
     ncCheck(nc_inq_grpname_full(myId,&lenp,charName),__FILE__,__LINE__);
     groupName = charName;
-    delete charName;
+    delete[] charName;
   }
   else {
     // return the (local) name of this group.
